x-pack/plugins/fleet/common/services/is_agent_upgradeable.test.ts

```diff
@@ -241,6 +242,154 @@ describe('Fleet - isAgentUpgradeable', () => {
   });
 });
 
+describe('Fleet - getNotUpgradeableMessage', () => {
```

Comment:
Going to make some wording suggestions on these error messages, and the test is the easiest place to make them all at once.

Reply:
Thanks! I added very basic messages but I appreciate having them a reviewed in a better english :D